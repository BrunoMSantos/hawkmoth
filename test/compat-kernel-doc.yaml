directive-arguments:
- compat-kernel-doc.c
directive-options:
  compat: kernel-doc
