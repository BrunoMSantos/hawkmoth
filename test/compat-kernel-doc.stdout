

@param quick brown fox
@return quick brown fox
@random quick brown fox


.. c:function:: int foo(int quick)

   
   @param quick brown fox
   @return quick brown fox
   @random quick brown fox
   

.. c:function:: int kerneldoc(int quick)

   
   
   :param quick: quick brown fox
   
   :return: quick brown fox
   
   :param random: quick brown fox
   

